main.go

```diff
@@ -55,11 +73,55 @@ func main() {
 		log.Err(err).Msg("failed to parse signing key")
 	}
 
-	config, err := shutterAPICommon.NewConfig(keyperHTTPUrl, signingKey)
+	p2pConfig := p2p.Config{}
+	p2pkey, err := keys.GenerateLibp2pPrivate(rand.Reader)
+	if err != nil {
+		log.Err(err).Msg("failed to generate p2p key")
+		panic(err)
+	}
+	p2pConfig.P2PKey = p2pkey
+
+	bootstrapAddressesStringified := os.Getenv("P2P_BOOTSTRAP_ADDRESSES")
+	if bootstrapAddressesStringified == "" {
+		panic("bootstrap addresses not provided in the env")
+	}
+	bootstrapAddresses := strings.Split(bootstrapAddressesStringified, ",")
+
+	bootstrapP2PAddresses := make([]*address.P2PAddress, len(bootstrapAddresses))
+
+	for i, addr := range bootstrapAddresses {
+		bootstrapP2PAddresses[i] = address.MustP2PAddress(addr)
+	}
+	p2pConfig.CustomBootstrapAddresses = bootstrapP2PAddresses
+	p2pConfig.ListenAddresses = []*address.P2PAddress{
+		address.MustP2PAddress("/ip4/0.0.0.0/tcp/23003"),
+		address.MustP2PAddress("/ip4/0.0.0.0/udp/23003/quic-v1"),
+		address.MustP2PAddress("/ip4/0.0.0.0/udp/23003/quic-v1/webtransport"),
+		address.MustP2PAddress("/ip6/::/tcp/23003"),
+		address.MustP2PAddress("/ip6/::/udp/23003/quic-v1"),
+		address.MustP2PAddress("/ip6/::/udp/23003/quic-v1/webtransport"),
```

Comment:
It's fine for now but eventually we should make the listen port number configurable.